### ISSUE TITLE
Separate ingredients into sections for readability

### ISSUE THREAD

**Issue by stehessel** [seanlane/gochowdown#27]
Hi,
It would be nice to somehow be able to separate long ingredient lists such that they become more readable. I know that I could use components for this but sometimes I don't want to clutter the components. An example would be to separate wet and dry ingredients in a baking recipe, different components don't really fit in that case. Something like

```
Ingredients:
[ ] apples
[ ] bananas

[ ] oranges
```

with new lines inbetween would be sufficient.

**Comment by Tiliavir** [comment]
I agree, that that would allow for some "light" component based recipes.

Guess that's not possible with empty lines in yaml, though. But something like the following could work:

```yaml
ingredients:
- Lasagna plates
- For the filling
  - tons of meat
- For béchamel
  - flour
  - butter
  - milk
```
